### ISSUE TITLE
.html syntax highlighting does not popnlock

### ISSUE THREAD

**Issue by Nuc1eoN** [Luxcium/pop-n-lock-theme-vscode#74]
The only reason why I don't use this theme compared to e.g. Moonlight 2 theme, is that the syntax highlighting is hard to parse for me while developing.

Pop N' Lock theme:
![popnlock](https://user-images.githubusercontent.com/2538022/83461388-ded0c780-a468-11ea-86a4-d7927e014996.PNG)
Moonlight 2 theme:
![m2](https://user-images.githubusercontent.com/2538022/83461391-e001f480-a468-11ea-8a99-7179b6dc2a33.PNG)

It would be cool to have a syntax highlighting that actually "pops and locks":D

**Comment by Luxcium** [comment]
can you please include code in the issue so I can try to replicate a screen capture exactly like yours

**Comment by Nuc1eoN** [comment]
Sure :smile:  Different code, but should have the same effect:
```html
<div ng-if="!$ctrl.isVisibleSeed" class="user-list get-started-content">

    <div class="margin-5 text-center">
        <div class="headline-1 margin-1" w-i18n="userList.exportUser"></div>
        <div class="body-2 margin-top-2 basic-500 margin-05">
            <span w-i18n="userList.notHaveAccount"></span>
        </div>
    </div>

    <div class="body-2 basic-900 margin-05" w-i18n="userList.address"></div>

    <w-select ng-model="$ctrl.activeUserAddress"
              one-item="$ctrl.userList.length <= 1"
              disabled="$ctrl.userList.length <= 1"
              hide-active="::false"
              class="select big margin-3">
        <w-option ng-repeat="user in $ctrl.userList track by user.address"
                  value="::user.address">

            <div class="avatar-container margin-right-1">
                <w-avatar type="{{user.userType}}"
                          class="avatar margin-right-1"
                          address="user.address"
                          size="30"></w-avatar>
            </div>

            <div ng-class="::{'has-name': !!user.name}" class="ellipsis">
                <div class="name body-2 margin-05 mobile-caption-2">{{::user.name}}</div>
                <div class="address footnote-1 mobile-caption-2 ellipsis">{{::user.address}}</div>
            </div>
        </w-option>
    </w-select>

    <div class="password-placeholder" ng-if="$ctrl.needPassword">
        <div class="body-2 basic-900 margin-05" w-i18n="userList.password"></div>
    </div>
    <form class="margin-2"
          name="$ctrl.loginForm"
          novalidate>

        <w-input-container class="margin-4" ng-if="$ctrl.needPassword">
            <w-show-hide-password>
                <w-input
                    w-autofocus
                    name="password"
                    type="password"
                    ng-model="$ctrl.password"
                    w-i18n-attr="placeholder"
                    class="big no-icon"
                    placeholder="userList.placeholders.password"
                    required>
                </w-input>
            </w-show-hide-password>
            <div class="like-input-error" ng-if="$ctrl.showPasswordError && !$ctrl.networkError">
                <span class="error active" w-i18n="password.error"></span>
            </div>
            <div class="like-input-error" ng-if="$ctrl.showPasswordError && $ctrl.networkError">
                <span class="error active" w-i18n="network.error"></span>
            </div>
        </w-input-container>

        <w-analytics event="'Migration Import Continue Click'" event-target="'ui'">
            <w-button disabled="$ctrl.loginForm.$invalid"
                      on-click="$ctrl.showSeed()"
                      type="submit"
                      class="submit big long">
                <span w-i18n="continue"></span>
            </w-button>
        </w-analytics>
    </form>

</div>

<div class="seed" ng-if="$ctrl.isVisibleSeed">
    <div class="backup-seed movable get-started-content">
        <div class="headline-1 margin-1 text-center">
            <span w-i18n="copySeed"></span>
        </div>

        <div class="body-2 basic-500 margin-top-1 margin-4 text-center">
            <span w-i18n="warn"></span>
            <a ng-click="$ctrl.backToUsersList()" class="submit-400" w-i18n="goBack"></a>
        </div>

        <div class="body-2 basic-900 margin-1">
            <span w-i18n="yourSeed"></span>
        </div>

        <w-seed-read type="read" seed="{{::$ctrl.seed}}" class="no-user-select margin-2 margin-top-1"></w-seed-read>

        <div class="margin-top-4">
            <a class="fake-button submit big long"
                      ng-click="$ctrl.clickCopySeed()"
                      w-copy="::$ctrl.seed">
                <span class="fake-button-content submit big long" w-i18n="copy"></span>
            </a>
        </div>
    </div>

</div>
```
